lib/loader.js

```diff
@@ -12,7 +11,21 @@ const pify = require("pify");
 // fs tasks when running the custom importer code.
 // This can be removed as soon as node-sass implements a fix for this.
 const threadPoolSize = process.env.UV_THREADPOOL_SIZE || 4;
-const asyncSassJobQueue = async.queue(sass.render, threadPoolSize - 1);
+let asyncSassJobQueue = null;
+let loadingError = null;
+
+try {
+    const sass = require("node-sass");
+    const sassVersion = require("node-sass/package.json").version;
+
+    if (/^4[.]/.test(sassVersion)) {
+        asyncSassJobQueue = async.queue(sass.render, threadPoolSize - 1);
+    } else {
+        loadingError = new Error("The installed version of \`node-sass\` is not compatible (expected: 4.x, actual: " + sassVersion + ").");
```

Comment:
`throw` here and catch the `{Error}` in (L27+) instead of an assignment please

Reply:
Good point. This will work better when we add support for things other than `node-sass` or something else bombs out.